utility/src/main/java/org/oppia/util/data/UrlImageParser.kt

```diff
@@ -0,0 +1,73 @@
+package org.oppia.util.data
```

Comment:
Repeat from an earlier PR: please send out a separate PR to submit this with its own tests. I'd like to review it outside the context of these other PRs, though it does mean you need to base these PRs on the branch introducing UrlImageParser.

Reply:
Sepearte PR [#201 ](https://github.com/oppia/oppia-android/pull/201) is been made for UrlImageParser